### DISCUSSION TITLE
Only showing PV1 AC out

### DISCUSSION THREAD

**Discussion by Chef-70** [robbinjanssen/home-assistant-omnik-inverter#136] [General]
Hello,

I  have the same Omnik omnik4000tl2 https://github.com/robbinjanssen/home-assistant-omnik-inverter/blob/master/images/config_flow.gif
I see PV1 and PV2 DC in but only PV1 AC out.
The output Wat of PV1 = to the output of the total inverter.
Is there something wrong with my inverter or is this by design?

Thanks,

Jef
![2022-10-20 13_56_45-Overzicht – Home Assistant en nog 4 andere pagina's - Privé - Microsoft​ Edge](https://user-images.githubusercontent.com/73892917/196942132-7aa99e7f-90ed-4298-8970-21e79bdc24c8.png)

**Comment by Chef-70** [comment]
Neverwind,

When i calculate DC in it says 546 W. Some loss so 485 W sounds okay.

**Comment by MarijnS95** [comment]
AC output isn't PV output (which outputs Direct Current :wink:), but the Alternating Current output from the inverter itself. The API supports up to 3 AC output channels though I doubt any inverter exists with that many.

**Comment by Funwar** [comment]
I know this topic is a bit old, But I am wondering if there has been a solution for the missing datapoints. My inverter only shows only three values ( actual power, energy produced today and total energy produced), So I would like to see some more info, therefore I'm interested in how your problem is solved, if so.

**Reply by robbinjanssen** [reply]
It depends on which datasource you use as well.. as far as i know only TCP provides these datapoints.
